Runtime/Scripts/MobiledgeXIntegrationConfig.cs

```diff
@@ -74,5 +79,50 @@ public void SetFallbackLocation(double longitude, double latitude)
             fallbackLocation.Longitude = longitude;
             fallbackLocation.Latitude = latitude;
         }
+
+        public static async Task<LocationFromIPAddress> GetLocationFromIP()
+        {
+            HttpClient httpClient = new HttpClient();
+            try
+            {
+                HttpResponseMessage response = await httpClient.GetAsync("https://freegeoip.app/json/").ConfigureAwait(false);
+                string responseBodyStr = response.Content.ReadAsStringAsync().Result;
+                LocationFromIPAddress location = Messaging<LocationFromIPAddress>.Deserialize(responseBodyStr);
+                return location;
+            }
+            catch (Exception)
+            {
+                return new LocationFromIPAddress()
+                {
+                    latitude = 37.3382f,
+                    longitude = 121.8863f
+                };
+            }
+        }
+
+        [DataContract]
+        public class LocationFromIPAddress
+        {
+            [DataMember]
+            public float longitude;
+            [DataMember]
+            public float latitude;
+        }
+
+        public static class Messaging<T>
```

Comment:
Do we have this class in c-sharp sdk? Seems useful and wouldn't need to duplicate code

Reply:
I tried to search for it, It's only in the PingPongGame.
@lgarner 
Do we have this Messaging class somewhere in the C# SDK?

Reply:
iOS has a custom deserializer, but this is useful code otherwise were it not for that.